rio_stac/stac.py

```diff
@@ -67,10 +67,17 @@ def get_projection_info(
     see: https://github.com/stac-extensions/projection/#item-properties-or-asset-fields
 
     """
-    try:
-        epsg = src_dst.crs.to_epsg() or None
-    except AttributeError:
+    projjson = None
+    if src_dst.crs is None:
         epsg = None
+    elif not src_dst.crs.is_epsg_code:
+        epsg = None
+        try:
+            projjson = src_dst.crs.to_dict(projjson=True)
+        except AttributeError:
+            projjson = None
+    else:
+        epsg = src_dst.crs.to_epsg()
```

Comment:
Can we do something like 👇  
```python
projjson = None
epsg = None
if src_dst.crs is not None:
    epsg = src_dst.crs.to_epsg() if src_dst.crs.is_epsg_code else None
    try:
        projjson = src_dst.crs.to_dict(projjson=True)
    except AttributeError:
        pass
```

Reply:
sure, thanks for the style improvement :)